x-pack/solutions/security/plugins/elastic_assistant/server/saved_objects.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { CoreSetup } from '@kbn/core/server';
+import { promptType } from './lib/prompt/saved_object_mappings';
+
+export const initSavedObjects = (savedObjects: CoreSetup['savedObjects']) => {
+  savedObjects.registerType(promptType);
```

Comment:
Given:

1) `initSavedObjects` is called during the `setup` phase of the plugin lifecycle
2) the implementation intends to fall back to reasonable defaults when the saved objects are unavailable

I'm wondering:

a) can `savedObjects.registerType(promptType)` fail (for example, with IO errors)?
b) if it can, should the plugin `setup` also fail?
c) if no, is it necessary for `savedObjects.registerType` be wrapped in a `try` / `catch`?